Makefile

```diff
@@ -1,121 +1,99 @@
-# Basys3 board using SymbiFlow, making it simple to understand
-# what to run with a straight-forward Makefile.
-#
-# (c) Henner Zeller <hzeller@google.com>
-# http://www.apache.org/licenses/LICENSE-2.0
-#
-# Set the environment variable SYMBIFLOW_ARCH_DEFS
-# to wherever the toplevel of your symbiflow-arch-defs
-# is checked out.
-#
-# Call
-#  make prog
-# To synthesize for Basys3 board.
-#
-# Current binaries needed in the process point to the versions
```

Comment:
I'd keep the readme at the top of the file, so that people can better understand what can be changed.

Reply:
(solved)